banderwagon/element_test.go

```diff
@@ -165,7 +165,9 @@ func TestSerde(t *testing.T) {
 
 	var buf bytes.Buffer
 
-	bandersnatch.WriteUncompressedPoint(&buf, &point_aff)
+	if _, err := bandersnatch.WriteUncompressedPoint(&buf, &point_aff); err != nil {
```

Comment:
You'll see many changes in the code that now aren't ignoring errors.
These changes aren't because I created new error scenarios: these errors were already there but just ignored! Now surfaced by `errcheck` linter.